tests/Agent/IntegrationTests/IntegrationTests/RemoteServiceFixtures/TracingChainFixture.cs

```diff
@@ -106,25 +106,6 @@ public RemoteApplication SetupReceiverApplication(bool isDistributedTracing, boo
             return receiverApplication;
         }
 
-        public HttpResponseHeaders ExecuteTraceRequestChainHttpWebRequest(IEnumerable<KeyValuePair<string, string>> headers)
```

Comment:
I'm assuming this was removed with some of the W3C chaining tests, but I'm not sure if we want this removed? Or if this should be migrated to a .net framework app?

Reply:
It was removed because `ExecuteTraceRequestChainHttpClient()` does the exact same thing and the other `XxxWebRequest` tests were removed as well.

Reply:
I think what's going on is that we had 2 different tests for distributed tracing where we chained multiple calls together. One test was meant to ensure that our HttpWebRequest instrumentation handled chaining correctly, and the other was meant to ensure that our HttpClient instrumentation handled chaining correctly. With the migration to HttpClient from HttpWebRequest, I think that we may have inadvertently removed our distributed tracing chaining test that was meant to validate our HttpWebRequest instrumentation.